backend/routes/api/index.js

```diff
@@ -4,14 +4,13 @@ const sessionRouter = require("./session.js");
 const usersRouter = require("./users.js");
 const myRouter = require("./my");
 const songRouter = require("./songs");
+const artistRouter = require("./artist");
```

Comment:
Consider prepending "/artists" to all of your artists route to DRY up your route paths